backend/src/main/java/com/bakdata/conquery/models/config/XodusStoreFactory.java

```diff
@@ -151,17 +183,17 @@ private void configureMapper(ConqueryConfig config) {
 	@Override
 	@SneakyThrows
 	public Collection<NamespaceStorage> loadNamespaceStorages() {
-		return loadNamespacedStores("dataset_", (elements) -> new NamespaceStorage(validator, this, elements));
+		return loadNamespacedStores("dataset_", (storesToTest) -> new NamespaceStorage(validator, this, storesToTest), NAMESPACE_STORES);
```

Comment:
warum toTest?

Reply:
Weil das die stores sind die wir erwarten, ich nenne es in `expectedStores` um ;)